lib/index.js

```diff
@@ -0,0 +1,37 @@
+module.exports = throttlingPlugin
+
+const Bottleneck = require('bottleneck/light')
+const ROUTES = require('@octokit/routes')
+const wrapRequest = require('./wrap-request')
+
+function buildRouteLookup (filter) {
+  return Object.values(ROUTES)
+    .reduce((acc, section) => {
+      section.filter(filter).forEach(route => { acc[route.path] = true })
+      return acc
+    }, {})
+}
+
+function throttlingPlugin (octokit) {
+  const state = {
+    triggersNotification: buildRouteLookup(route => route.triggersNotification),
+    minimumAbuseRetryAfter: 5,
+    retries: 1,
+    globalLimiter: new Bottleneck({
+      maxConcurrent: 1
+    }),
+    writeLimiter: new Bottleneck({
+      minTime: 1000
+    }),
+    triggersNotificationLimiter: new Bottleneck({
+      minTime: 5000
```

Comment:
Lets set it to 3000s. That's what we used with ended up using for [semantic-release](https://github.com/semantic-release/github/blob/761d39ebb89bcf3f4198755905a258fc95388d94/lib/definitions/rate-limit.js#L16) which creates releases and issues. I cannot recall a single report of someone hitting an abuse limit, and at times people have hundreds of issues/PRs that receive comments for a new release

Reply:
Already done in a later commit!